src/ServiceControl.Monitoring.SmokeTests.AzureStorageQueues/packages.config

```diff
@@ -7,7 +7,7 @@
   <package id="Newtonsoft.Json" version="12.0.1" targetFramework="net461" />
   <package id="NServiceBus" version="7.1.6" targetFramework="net461" />
   <package id="NServiceBus.AcceptanceTesting" version="7.1.6" targetFramework="net461" />
-  <package id="NServiceBus.Azure.Transports.WindowsAzureStorageQueues" version="8.1.0" targetFramework="net461" />
```

Comment:
that explains why the customer see huge ASQ spikes

Reply:
This is the smoke test reference. I think it was missed when doing an update to 8.1.1.